src/cards_management.c

```diff
@@ -66,16 +68,24 @@ int initialize_cards(void)
     }
     g_draw_pile->next = NULL;
 
-    for (i = RED; i <= YELLOW; i++)
+	int count_wild_card = 0;
+    for (i = RED; i <= BLACK; i++)
     {
+    	if(i == BLACK && count_wild_card != 2)
```

Comment:
Please following the Coding conventions in the project requirement to cleanup all of your codd.